custom.build.mjs

```diff
@@ -1,27 +1,63 @@
 import tsFileStruct from "ts-file-parser";
 import fs from "fs";
 import { exec } from "child_process";
-let classMappings = {
-	Page: { name: "Page" },
-	Application: { name: "Application" },
-	Client: { name: "Client" },
-	Component: { name: "Component" },
-	Form: { name: "Form" },
-	Table: { name: "Table" },
-	TableForm: { name: "TableForm" },
-	Popup: { name: "Popup" },
-	Formatter: { name: "Formatter" },
-	LowcodeSDK: { name: "kf", staticDeclarations: true }
+
+const LIB_KIND = {
```

Comment:
LIBRARY_KIND

Reply:
Changed the name into SDK_KIND since its more readable than LIBRARY_KIND.